frontend/services/screenshot_service/Screenshot_Service.hpp

```diff
@@ -63,6 +66,10 @@ class Screenshot_Service final : public AbstractFrontendService {
     std::vector<FrontendResource> m_providedResourceReferences;
     std::vector<std::string> m_requestedResourcesNames;
     std::vector<FrontendResource> m_requestedResourceReferences;
+
+    /// XXX TEMP
+    bool guiRenderCallback(WinConfig_t& wc);
+    megamol::frontend_resources::GUIRenderRequest m_guiRenderRequest_resource;
```

Comment:
It would be nice if we could simply register the GUI Render Request via a Lambda and not hold an additional resource in our service that we need to expose. For example, 
```frontend_resources[gui_window_render_callbacks_resource].add_request( 
    {window_name, 
    [&](WinConfig_t& wc) {
    ... 
} } )
```

Reply:
see commit b3fe3c4